src/core/stream.c

```diff
@@ -659,6 +661,23 @@ QuicStreamParamSet(
         break;
     }
 
+   case QUIC_PARAM_STREAM_RELIABLE_OFFSET:
+
+        if (BufferLength != sizeof(uint64_t) || Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+
+        if (!Stream->Connection->State.ReliableResetStreamNegotiated ||
+            *(uint64_t*)Buffer > Stream->QueuedSendOffset) {
+            Status = QUIC_STATUS_INVALID_STATE;
+            break;
+        }
+
+        Stream->ReliableOffsetSend = *(uint64_t*)Buffer;
```

Comment:
We should add a log too. Something like:
```c
QuicTraceLogStreamInfo(
    ReliableSendOffsetSet,
    Stream,
    "Reliable send offset set to %llu",
    *(uint64_t*)Buffer);
```

Reply:
Added